.pre-commit-config.yaml

```
@@ -39,11 +39,11 @@ repos:
     hooks:
       - id: add-trailing-comma
   - repo: https://github.com/pycqa/docformatter
-    rev: "eb1df347edd128b30cd3368dddc3aa65edcfac38" # v1.7.5 with pre-commit fix; awaiting updated tag
+    rev: "v1.7.5" # v1.7.5 with pre-commit fix; awaiting updated tag

```

Really wish pre-commit.ci would stop trying to change this one:
```suggestion
    rev: "eb1df347edd128b30cd3368dddc3aa65edcfac38" # v1.7.5 with pre-commit fix; awaiting updated tag
```